src/security.cs

```diff
@@ -1103,7 +1105,6 @@ interface SecKeyKeyExchangeParameter {
 		NSData SharedInfo { get; set; }
 	}
 
-#if IOS || MONOMAC
 	[iOS (8,0)][NoTV][NoWatch]
```

Comment:
Same here.

Reply:
This was already included in MACCATALYST because it is inside a IOS block, I'd say that we probably have it already in our min version of catalyst. Probably we want to be explicit.